src/consensus/tx_verify.cpp

```diff
@@ -150,27 +150,33 @@ template unsigned int GetP2SHSigOpCount<const Coin>(
 template unsigned int GetP2SHSigOpCount<std::reference_wrapper<const Coin>>(
     const CTransaction& tx, const std::span<std::reference_wrapper<const Coin>>);
 
```

Comment:
In "consensus: Use Coin span in GetTransactionSigOpCost" 3bdb4a89a9e852ddee9cebaaf3492473b16e7cc6

The last paragraph in the commit message is not stale from prev commit?